### PR TITLE
build: add build script foundation for packages

### LINKED ISSUE #11: Build @carbon/ibm-cloud-paks npm module

### PR DESCRIPTION
**Pull request by SimonFinney**
Closes #11 

**Request for comments -** Adds basic build script foundation for packages currently in use using common rollup and Sass configuration currently available.

#### Changelog

**New**

- Add JavaScript (via rollup) and Sass (via `@carbon/bundler`) build scripts to `cloud-paks`, `experimental-package`, and `security` packages
- Add `files` property in relevant `package.json`s to indicate build artifacts
- Add `test` scripts to verify Sass compilation
- Add SCSS output to `.gitignore`

**Changed**

Move shared dependencies to root `package.json`

**Removed**

- Remove `example-component-package` as this has been superseded by `experimental-package`
- Remove `~` in Sass files causing compilation errors
- Remove Storybook and webpack development dependencies from `experimental-package` as these are available in `core` - @lee-chase to confirm if this is correct

### CODE REVIEW

**Comment by SimonFinney** [comment]
@carbon-design-system/ibm-cloud-paks-development As mentioned above this is a request for comments, so please add any general thoughts via the thread here, or as individual line comments for anything specific - Thank you!

**Review by dbrugger — APPROVED** [review]
lgtm

File: packages/cloud-paks/README.md [hunk 1]
```diff
@@ -4,26 +4,26 @@
 
 ## Getting started
 
-To install `@carbon/ibm-cloud-paks` in your project, you will need to run the
-following command using [npm](https://www.npmjs.com/):
+To install `@carbon/ibm-cloud-paks` in your project, you'll need to run one of
```
Comment (lee-chase) [inline packages/cloud-paks/README.md:7]: Concerned that we are creeping closer to accidentally publishing a package called ibm-cloud-paks. I am tempted just to go with 'carbide-components' and see if anyone higher up complains.

**Review by lee-chase — APPROVED** [review]
No obvious issues with this. I worry that we are creeping closer to not having a better repository/package name but other than that it looks fine.

**Pull request merged**